File: user-center/src/main/java/org/unreal/cloud/user/dao/BaseUserDao.java
package org.unreal.cloud.user.dao;

import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;
import org.unreal.cloud.user.model.BaseUserEntity;

@Repository
public interface BaseUserDao extends JpaRepository<BaseUserEntity , Integer> {
}

File: user-center/src/main/kotlin/org/unreal/cloud/user/dao/BaseUserDao.kt
package org.unreal.cloud.user.dao

import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository
import org.unreal.cloud.user.model.BaseUserEntity

@Repository
interface BaseUserDao : JpaRepository<BaseUserEntity, Int> {
    fun findByAccountAndPassword(account: String, password: String): BaseUserEntity?
    fun countByAccount(account: String): Int
}
